includes/datavalues/SMW_DV_Number.php

```diff
@@ -71,10 +71,20 @@ static protected function parseNumberValue( $value, &$number, &$unit ) {
 		$decseparator = NumberFormatter::getInstance()->getDecimalSeparatorForContentLanguage();
 		$kiloseparator = NumberFormatter::getInstance()->getThousandsSeparatorForContentLanguage();
```

Comment:
Note that the regex below can explode if one of these delimiters is longer than 1 character.